src/renderer/pages/toast/index.js

```
@@ -1708,6 +1726,17 @@ function executeButton(button) {
   // Execute button action in normal mode
   showStatus('Executing...', 'info');
 
+  // 특별 명령어 처리 (꽃가루 애니메이션)
+  if (button.action === 'script' && button.scriptType === 'special' && button.script === 'confetti') {
+    // 꽃가루 애니메이션 실행
+    showStatus('🎉 Let it go!', 'success');
+    window.confetti.start({
+      duration: 5,  // 5초 동안 실행
+      density: 100  // 꽃가루 밀도
```

ESLint

[error] Missing trailing comma.

(comma-dangle)